### PR TITLE
Add Batch export layers script

### PR DESCRIPTION
**Pull request by mdouchin**

### CODE REVIEW

File: scripts/Batch_export_layers.py [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/mdouchin/.qgis2/processing/scripts/Batch_export_layers.py
```
Comment (luipir) [inline scripts/Batch_export_layers.py]: ???

File: scripts/Batch_export_layers.py.help [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/mdouchin/.qgis2/processing/scripts/Batch_export_layers.py.help
```
Comment (luipir) [inline scripts/Batch_export_layers.py.help]: ???

**Comment by mdouchin** [comment]
Arg, symbolic links :(
I will correct this asap !

File: scripts/Batch_export_layers.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    # Create filename for this layer
+    output_name = layer.name()
+    if layer.providerType() == 'postgres' and Use_table_name_instead_of_layer_name_for_database_layers:
+        output_name = QgsDataSourceURI(layer.dataProvider().dataSourceUri()).table()
+    output_filename = Output_folder + '/' + output_name + '.' + ogr_extensions[Format]
+    progress.setInfo(output_filename)
+
+    # CRS
+    lcrs = layer.crs()
+    progress.setInfo('Layer CRS = %s' % lcrs.authid())
```
Comment (luipir) [inline scripts/Batch_export_layers.py]: btw are you sure this script is for master version... seems more for 2.18? progress has been substituted by feedback? btw I don't have so much experience in creating processing scripts to judge.
A good way to check is to add unit test for this script... as usual ,any new feature, like this have to be followed by unit test.

Comment (mdouchin) [reply scripts/Batch_export_layers.py]: Indeed it is for QGIS 2. In processing 3, script must be written like algs, with the processing class and methods cf https://anitagraser.com/2018/03/25/processing-script-template-for-qgis3/
My mistake, sorry. I will propose a new PR against qgis-2 branch

**Comment by mdouchin** [comment]
@luipir new PR against qgis-2 Thanks in advance

**Pull request closed without merging**